src/main/java/net/sf/jabref/gui/maintable/MainTableSelectionListener.java

```diff
@@ -105,25 +105,16 @@ public void listChanged(ListEvent<BibEntry> e) {
         if (!enabled) {
             return;
         }
+
         EventList<BibEntry> selected = e.getSourceList();
-        BibEntry newSelected = null;
-        while (e.next()) {
-            if (e.getType() == ListEvent.INSERT) {
```

Comment:
What happens if an entry is inserted? I have no idea how the code works, but it seems like this is some sort of handling for inserted entries?